model/distro/distro.go

```diff
@@ -365,6 +366,13 @@ func (d *Distro) GetMainlineTimeInQueueFactor() int64 {
 	return d.PlannerSettings.MainlineTimeInQueueFactor
 }
 
+func (d *Distro) GetStepbackTaskFactor() int64 {
```

Comment:
we can get rid of this / remove d.PlannerSettings.StepbackTaskFactor

Reply:
done

Reply:
not done